src/main/java/com/minecolonies/coremod/colony/Colony.java

```diff
@@ -635,7 +641,8 @@ else if (!isDay && world.isDaytime())
         if(this.isDirty && shallUpdate(world, CLEANUP_TICK_INCREMENT))
         {
             this.isDirty = false;
-            ColonyManager.markDirty();
```

Comment:
don't we need this call to update clients?

Reply:
not anymore =)